fuse/redhat731/istio-distributed-tracing/booster.yaml

```diff
@@ -0,0 +1,20 @@
+name: Red Hat Fuse - Istio - Distributed Tracing
```

Comment:
There is a PR asking to remove the ISTIO examples: https://github.com/fabric8-launcher/launcher-booster-catalog/pull/354

Maybe this can be left out for now?